src/main/cpp/utils/TargetTracker.cpp

```diff
@@ -11,9 +11,35 @@
 TargetTracker::TargetTracker(TargetTrackerConfig config,
                              IntakeSubsystem* intake, ScoringSubsystem* scoring,
                              DriveSubsystem* drive)
-    : m_config{config}, m_intake{intake}, m_scoring{scoring}, m_drive{drive} {}
+    : m_config{config}, m_intake{intake}, m_scoring{scoring}, m_drive{drive} {
+  m_trackedTargets = std::vector<TrackedTarget>(m_config.maxTrackedItems);
+}
 
 std::vector<DetectedObject> TargetTracker::GetTargets() {
+  if (!frc::RobotBase::IsReal()) {
+    static int counter = 0;
+    return {
+        DetectedObject(1, 0.75,
```

Comment:
Now that we're doing more "sim only" stuff like this you should have constants for these (maybe a namespace?) (maybe unit tests using them?)

Reply:
agreed

Reply:
Let's do this in the cleanup branch this week